webapp/src/octoClient.ts

```diff
@@ -75,8 +75,18 @@ class OctoClient {
         return false
     }
 
-    logout(): void {
+    async logout(): Promise<boolean> {
+        const path = '/api/v1/logout'
+        const response = await fetch(this.getBaseURL() + path, {
+            method: 'POST',
+            headers: this.headers(),
+        })
         localStorage.removeItem('focalboardSessionId')
```

Comment:
Probably we should delete this only if the logout API call is a success, but we would need to notify that to the user somehow. This is the initial approach, but we should make it more explicit, so if the session is not removed from the server (for any reason) we don't consider you logged out, and we just show you an error message. But that would be needed to be done as another task.

Reply:
IMO we should warn if user was not completely logged out, but still remove the local cookie.  This could be someone stepping away from a public computer and we should not be leaving a cookie there when they explicitly asked us not to.